src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -369,6 +392,34 @@ private Channel createChannel() {
             channel.basicQos(configuration.getPrefetchCount());
             channel.addReturnListener(ret ->
                     LOGGER.warn("Can not router message to exchange '{}', routing key '{}'. Reply code '{}' and text = {}", ret.getExchange(), ret.getRoutingKey(), ret.getReplyCode(), ret.getReplyText()));
+            if(isPublisher) {
+                channel.confirmSelect();
+                channel.addConfirmListener(
+                        (deliveryTag, multiple) -> {
+                            if(LOGGER.isTraceEnabled()) {
+                                ChannelHolder channel1 = channelsByPin.get(pinId);
+                                String sequence = "null";
+                                if(channel1 != null) {
+                                    sequence = channel1.outstandingConfirms.get(deliveryTag);
+                                }
+                                if(!multiple) {
+                                    LOGGER.trace("Message with delivery tag {} confirmed. Message: {}", deliveryTag, sequence);
+                                } else {
+                                    LOGGER.trace("Messages prior to delivery tag {} confirmed. Message: {}", deliveryTag, sequence);
+                                }
+                            }
+                        },
+                        (deliveryTag, multiple) -> {
```

Comment:
Currently common doesn't resend batches, could you please remove messages from cache